exercises/concept/chessboard/.docs/introduction.md

```diff
@@ -92,3 +91,33 @@ for range xi {
 // count value:
 // 3
 ```
+
+You've previously seen defining struct types, but it's also possible to define non
```

Comment:
Prefer to put line breaks at sentence breaks. In both the actual display and the review display for this the lines wrap and doing it like this just increases the blast radius for lines changed when we do reviews. For example, removing or changing one or two words ends up meaning you have to reformat the whole paragraph.

Reply:
nit: "non" is not a word on its own, so hyphenate: non-struct